sig-node/CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing on SIG Node
+
+Welcome!
+
+## For Kubernetes Contributions
+
+https://github.com/kubernetes/community/tree/master/contributors/guide#contributor-guide
+
+### For Enhancements 
+
+https://github.com/kubernetes/enhancements/tree/master/keps/sig-node
+
+#### Helpful Links for Sig-Node
+
+Code:
```

Comment:
as minimum, probe folder is also worth mentioning:

kubernetes/pkg/probe/
kubernetes/vendor/k8s.io/kubelet/
kubernetes/vendor/k8s.io/cri-api

Reply:
Added